packages/core/src/mls/MLSService/MLSService.ts

```diff
@@ -95,7 +95,7 @@ export class MLSService {
   public configureMLSCallbacks({groupIdFromConversationId, ...coreCryptoCallbacks}: MLSCallbacks): void {
     this.coreCryptoClient.registerCallbacks({
       ...coreCryptoCallbacks,
-      clientIdBelongsToOneOf: (client, otherClients) => {
+      clientIsExistingGroupUser: (client, otherClients) => {
```

Comment:
it was renamed in new version of core-crypto

Reply:
you might need to defined the `authorize` callback also :/